p4-e.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="ufl-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Tresure-Hunt</title>
+    <meta name="viewport" content="width=device-width, intial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="p4-e.css" />
+    <script src="p4-e.js"></script>
+</head>
+
+<body>
+    <header></header>
+    <main>
+        <h1>暗号</h1>
+        <div></div>
+        <div>00 00 00 00</div>
+    </main>
+    <footer></footer>
+</body>
+
+</html>
```

Comment:
Refs: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

```suggestion
</html>

```